README.rst

```diff
@@ -65,22 +65,28 @@ Getting started
 --------------------------------------------------------------------------------
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Create your first application
+Prerequisites
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To get a template application that uses Tarantool Cartridge and run it,
-you need to install the `cartridge-cli` utility (supposing that
-`Tarantool <https://www.tarantool.io/en/download/>`_ is already installed).
+you need to install several packages:
+
+* ``tarantool``, ``tarantool-dev``, ``cartridge-cli``
```

Comment:
let's move cartridge-cli to a standalone bullet item and provide a link to https://github.com/tarantool/cartridge-cli#installation